Specs/Number/Japanese/NumberModel.json

```diff
@@ -3931,7 +3932,9 @@
   {
     "Input": "この袋の厚さは0.08ミリメートルです。",
     "IgnoreResolution": true,
+    "Comment": "Need language review",
     "NotSupportedByDesign": "javascript, java, python",
+    "NotSupported": "dotnet",
```

Comment:
It was passing previously and broke in this PR?

Reply:
After review we realized that this should be negative test cases. That's why it's marked as notsupported for now. We'll be reviewing again with language expert.